src/carma-streets/plot_message_frequencies.py

```diff
@@ -5,6 +5,7 @@
 import pandas as pd
 from matplotlib import axes
 from matplotlib import pyplot as plt
+from measurement_time_metric import MeasurementTimeMetric
```

Comment:
Hey Dan, since this is a different plot I think we should not call it from the plot_message_frequency script. I think you can probably copy some of the arg stuff from the plot_message_frequency script (only what makes sense of course since you don't need a sim flag or message types or whatever). Then you can create a script that is plot_sdsm_detection_delay.